stdlib/public/core/AutoDiff.swift

```diff
@@ -336,6 +336,34 @@ public extension Differentiable {
 // Free-function-style differential operators
 //===----------------------------------------------------------------------===//
 
+// Transpose
+
+@inlinable
```

Comment:
Since these operators are not usable, please add a `@available` attribute to each of the functions you added to make them produce a compile-time error upon use.
```suggestion
@available(*, unavailable)
@inlinable
```